packages/debug/package.json

```diff
@@ -60,7 +60,9 @@
     "docs": "theiaext docs"
   },
   "devDependencies": {
-    "@theia/ext-scripts": "^0.3.18"
+    "@theia/ext-scripts": "^0.3.18",
+    "mkdirp": "^0.5.0",
```

Comment:
they should go to `dependencies` otherwise they will be missing when clients execute `download-adapters.js`

Reply:
Nice catch, although these are used only in development, I can see how dependant libraries wouldn't see them

Reply:
Fixed

Reply:
> although these are used only in development

They are used on postinstall npm hook by clients. Adapters are not distributed together with extensions.